### PR TITLE
feat: 131 add redirect field

### PR DESCRIPTION
**Pull request by jcpitre**
Closes https://github.com/MobilityData/mobility-database-catalogs/issues/131

Added redirects to the json schema.
Modified export to csv script to process redirects.

This PR has an associated issue in mobility-feed-api: https://github.com/MobilityData/mobility-feed-api/issues/149
And PR: https://github.com/MobilityData/mobility-feed-api/pull/150

### CODE REVIEW

File: .github/workflows/export_to_csv.yml [hunk 1]
```diff
@@ -129,6 +152,9 @@ jobs:
   store-csv:
     needs: [ export-to-csv ]
     runs-on: ubuntu-latest
+    # Don't upload to Google Cloud if it was triggered by hand.
+    # In that case the resulting sources.csv can be found in the build artifacts.
+    if: ${{ github.event_name != 'workflow_dispatch' }}
```
Comment (davidgamez) [inline .github/workflows/export_to_csv.yml:157]: question: There is no harm uploading the artifact to Google cloud. Why prevent it?

Comment (jcpitre) [reply .github/workflows/export_to_csv.yml:157]: I added the workflow_dispatch as a way to test the creation of the csv. I don't feel comfortable uploading it each time someone presses the button. But I could remove the restriction if you feel it's ok.

**Review by davidgamez — APPROVED** [review]
LGTM!

**Review by emmambd — APPROVED** [review]
LGTM!

**Pull request merged**